package.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
```

Comment:
I think this script can be deleted but you'll have to delete lines 71 and 72 in projenrc

Reply:
Done.